src/api/spec/controllers/webui/architectures_controller_spec.rb

```diff
@@ -6,10 +6,17 @@
 
   it { is_expected.to use_before_action(:require_admin) }
 
-  describe 'POST #bulk_update_availability' do
-    context 'as admin' do
-      it 'creates the architectures' do
-        login(admin_user)
+  before do
+    login(admin_user)
+  end
+
+  describe 'GET #index' do
+    before do
+      get :index
+    end
+
+    it { expect(assigns(:architectures)).to match_array(Architecture.all) }
+  end
 
         post :bulk_update_availability, params: { archs: { i586: '0', s390x: '1' } }
         expect(response).to redirect_to(architectures_path)
```

Comment:
Looks like it get's fixed in ec6f8318691b330fb1ff7d64b9eefa84a91f68ed . But would be nice to do that directly in 59ed5471b174da2e886b838475123b7212163780 , eg. via git rebase -i 59ed5471b174da2e886b838475123b7212163780^